contents/docs/feature-flags/common-questions.mdx

```diff
@@ -43,6 +43,39 @@ import FAQFalseOrNoneEvents from "./snippets/faq-false-or-none-events.mdx"
 
 Evaluating feature flags requires making a request to PostHog. However, you can [evaluate feature flags locally](/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation) instead – without having to make a request to PostHog.
 
+
+## How can I estimate the number of feature flag requests I'll be charged for?
+
+### Frontend SDKs
+
+We make a request to fetch feature flags when:
+
+1. The PostHog SDK is initialized
+2. A user is [identified](/docs/data/identify) 
+3. A user's [properties](/docs/product-analytics/user-properties) are updated.
+4. You ask us to reload feature flags (`posthog.reloadFeatureFlags()`)
```

Comment:
I think this is a section separate from estimating. It should be an h2 above and leave this section as just the `/decide` estimation. Knowing this doesn't help you estimate, but it is useful.